src/renderer/components/ProgramChangeView.tsx

```diff
@@ -70,6 +80,23 @@ function ProgramChangeView() {
     displayList.push(key);
     pinnedAccount[key] = true;
   });
+
+  function sortFunctionDUP(a: AccountInfo, b: AccountInfo) {
```

Comment:
What's `DUP` stand for? Down, up?

Reply:
na, lazy naming - I've slept on this, and think i need to go deeper.